src/ontology/imports/merged_import.owl

```diff
@@ -116500,6 +116537,7 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/PR_000001006> "PR:000001006")
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/PR_000001006> "Category=gene. Requested by=CL.")
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/PR_000001006> "receptor-type tyrosine-protein phosphatase C")
+AnnotationAssertion(<ttps://w3id.org/semapv/vocab/crossSpeciesExactMatch> <http://purl.obolibrary.org/obo/PR_000001006> <http://purl.obolibrary.org/obo/PR_P08575>)
```

Comment:
Something wrong happened here: the IRI for the annotation property should be `https://w3id.org/semapv/vocab/crossSpeciesExactMatch`, not `ttps://w3id.org/semapv/vocab/crossSpeciesExactMatch`.

This is probably an error in the upstream import, so I won’t block the PR for that, but this error must be noted. I’ll try tracking the provenance of the error.

Reply:
Thanks for investigating, @gouttegd.

Reply:
FYI: the error came from the PR slim (https://github.com/obophenotype/pro_obo_slim/issues/32). It’s fixed now, so the next time the imports are refreshed those incorrect IRIs will go away automatically.

Reply:
Thank you, @gouttegd. Unless advised to do sooner, the next import refresh will happen after the July release and before the August release.

Reply:
If the issue is already fixed, I suggest refreshing the imports again today not to propagate the issue.

Reply:
I agree. I don’t like the idea of making a release with known errors in it.